db/migrate/20220804161146_add_service_id_to_services.rb

```diff
@@ -0,0 +1,5 @@
+class AddServiceIdToServices < ActiveRecord::Migration[7.0]
+  def change
+    add_column :services, :service_id, :bigint
```

Comment:
this seems to be redundant to add a service ID to the service table. and I dont think we will be using this column

Reply:
when developing the code only worked when i added service_id, I was gonna ask you about the logic behind it. 

Before adding service_id, the error was Attribute Missing.

Reply:
done